core/src/cluster_info_vote_listener.rs

```diff
@@ -561,12 +598,23 @@ impl ClusterInfoVoteListener {
                     slot,
                 );
                 w_slot_tracker.total_stake += current_stake;
+                w_slot_tracker.gossip_only_stake += gossip_only_stake
```

Comment:
I think we should add a test to verify this tracker value is correct. It looks like we don't retain gossip votes for pubkeys for which we have already received replay votes for, so I don't think it will count all the gossip votes.

Reply:
@jstarry ah yeah, good catch, will fix and add test!

Reply:
Fixed and tests added!